src/main/java/Parser.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Parser class interprets the user's command.
+ */
+public class Parser {
+
+    /**
+     * Parses the input command given by the user to determine which functionality to execute.
+     *
+     * @param input String entered by the user, denoting instructions for the program.
+     */
+    public static void parseInput(String input) {
+        String[] inputParts = input.split(" ");
+        String instruction = inputParts[0];
+
+        switch (instruction) {
+        case "done":
+            TaskList.markComplete(Integer.parseInt(inputParts[1]), true);
+            break;
+        case "bye":
+            Ui.bye();
+            break;
+        case "list":
+            TaskList.list();
+            break;
+        case "delete":
+            TaskList.deleteTask(Integer.parseInt(inputParts[1]));
+            break;
+        case "find":
+            TaskList.findTask(inputParts[1]);
+            break;
+        default:
+            attemptStore(input);
+            break;
+        }
+    }
+
+    /**
+     * Attempts the store feature on the given user input. If the input does not correspond to todo, deadline or event,
+     * a DukeException is caught. This signifies a wrong input command or an incomplete description of the given task.
+     *
+     * @param input Input command given by the user
+     */
+    private static void attemptStore(String input) {
```

Comment:
Perhaps `store` is a better name. Now as a reader I get this question: Ok attempted to store, so where is exactly store happening?
Exception is a mechanism to say failed storing. Hope you get the drift?